src/cr/cube/cubepart.py

```diff
@@ -601,7 +610,7 @@ def scale_means_rows_margin(self):
 
     @lazyproperty
     def scale_median_column(self):
-        """ -> np.int64 ndarray of the columns scale median
+        """-> np.int64 ndarray of the columns scale median
```

Comment:
We tried out this ` -> ` shorthand for `"Returns ..."` because it was compact and consistent with Python3 type-hints. But the new version of `Black` won't allow the leading space and then the eye struggles to differentiate it from the `"""`. So we're back to using `Returns` when applicable (methods and functions). This is a property though, so the arrow should just go away, leaving `"""np.int64 ndarray of ..."""`.